src/robot/libraries/DateTime.py

```diff
@@ -325,13 +326,18 @@
     "subtract_time_from_time",
 ]
 
+DateOutput = Union[datetime.datetime, float, str]
+DateInput = Union[datetime.datetime, datetime.date, float, int, str]
+TimeInput = Union[datetime.timedelta, float, int, str]
```

Comment:
`int` is most likely unnecessary here, because accepting `float` means that values like string `"2"` and integer `3` are accepted and will be converted to floats `2.0` and `3.0`, respectively. I'll test that locally after merging this PR and also look how this shows up in Libdoc output.

Reply:
I tested this locally and `int` is unnecessary. In the generated documentation listing it along with `float` makes it more explicit that integers are accepted, though, so I guess we can leave it there.